paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```
@@ -378,15 +410,30 @@ internal sealed interface PaymentSheetScreen {
             }
         }
 
-        override fun title(isCompleteFlow: Boolean, isWalletEnabled: Boolean): StateFlow<ResolvableString?> {
+        override fun title(isCompleteFlow: Boolean, isWalletEnabled: Boolean): ResolvableString? {
+            // Should never be used
+            // headerText of this screen is set in paymentSheetScreenContentState
+            return interactor.state.value.title
+        }
```

Is there a better way to avoid calling this function? I don't think we want to throw an error though.